package.json

```diff
@@ -188,7 +188,7 @@
     "rollup-plugin-angular-aot": "0.0.4",
     "rollup": "0.41.4",
     "run-sequence": "1.2.2",
-    "sass-loader": "6.0.2",
+    "sass-loader": "6.0.3",
```

Comment:
At this point all the projects are in sync on library version numbers. If you need this update, then we should apply it to all the other libraries we are working with.